### PR TITLE
habib/add update t&c

### PR DESCRIPTION
**Pull request by ShuvoHabib**
TnC update for ASIC restriction

### CODE REVIEW

**Review by kevinw-binary — COMMENTED** [review]
we need to revert the changes on `package-lock.json`

File: src/translations/ach_UG.po [hunk 1]
```diff
@@ -10966,7 +10966,7 @@ msgstr "crwdns68134:0crwdne68134:0"
...
+msgid "The Company has a Stop Out policy which prevents the client from losing more than they have deposited. However, in the event that the client's Open Positions are not closed when the Margin Level for their MT5 Real Account reaches the Stop Out Level, the Company, at its sole discretion, may waive the client's negative balance by crediting their account if their Account Balance goes into a negative balance. The negative balance is determined by aggregating all the negative balances incurred over a 24-hour period across all accounts held by the client. The client acknowledges that, unless they are a retail customer of Deriv Investments (Europe) Limited, a client of Deriv (Europe) Limited, or a retail customer residing in Australia, the offer of the negative balance protection by the Company to the client is at the Company's sole discretion and the Company reserves the right to change the features and eligibility criteria of the negative balance protection at any time. The provisions of the negative balance protection do not apply in the following situations:"
```
Comment (kevinw-binary) [inline src/translations/ach_UG.po:10969]: I think we can revert these changes on translations

since we do translation after the PR is ready and merged ( or staging release)

Comment (ShuvoHabib) [reply src/translations/ach_UG.po]: Thanks @kevinw-binary , Resolved

**Comment by ShuvoHabib** [comment]
> we need to revert the changes on `package-lock.json`

Resolved

File: src/templates/static/legal/metatrader/terms_and_conditions.jsx [hunk 2]
```diff
@@ -205,7 +205,7 @@ const TermsAndConditions = () => (
...
+                <p>{it.L('The Company has a Stop Out policy which prevents the client from losing more than they have deposited. However, in the event that the client\'s Open Positions are not closed when the Margin Level for their MT5 Real Account reaches the Stop Out Level, the Company, at its sole discretion, may waive the client\'s negative balance by crediting their account if their Account Balance goes into a negative balance. The negative balance is determined by aggregating all the negative balances incurred over a 24-hour period across all accounts held by the client. The client acknowledges that, unless they are a retail customer of Deriv Investments (Europe)  Limited, a client of Deriv (Europe) Limited, or a retail customer residing in Australia, the offer of the negative balance protection by the Company to the client is at the Company\'s sole discretion and the Company reserves the right to change the features and eligibility criteria of the negative balance protection at any time. The provisions of the negative balance protection do not apply in the following situations:')}
```
Comment (negarn) [inline src/templates/static/legal/metatrader/terms_and_conditions.jsx:208]:
```suggestion
                <p>{it.L('The Company has a Stop Out policy which prevents the client from losing more than they have deposited. However, in the event that the client\'s Open Positions are not closed when the Margin Level for their MT5 Real Account reaches the Stop Out Level, the Company, at its sole discretion, may waive the client\'s negative balance by crediting their account if their Account Balance goes into a negative balance. The negative balance is determined by aggregating all the negative balances incurred over a 24-hour period across all accounts held by the client. The client acknowledges that, unless they are a retail customer of Deriv Investments (Europe) Limited, a client of Deriv (Europe) Limited, or a retail customer residing in Australia, the offer of the negative balance protection by the Company to the client is at the Company\'s sole discretion and the Company reserves the right to change the features and eligibility criteria of the negative balance protection at any time. The provisions of the negative balance protection do not apply in the following situations:')}
```

Comment (ShuvoHabib) [reply src/templates/static/legal/metatrader/terms_and_conditions.jsx]: Thanks, resolved

File: src/templates/static/legal/metatrader/terms_and_conditions.jsx [hunk 3]
```diff
@@ -205,7 +205,7 @@ const TermsAndConditions = () => (
...
                     <ol>
```
Comment (ShuvoHabib) [inline src/templates/static/legal/metatrader/terms_and_conditions.jsx:209]: Add suggestion commit

File: src/templates/static/legal/metatrader/terms_and_conditions.jsx [hunk 4]
```diff
@@ -205,7 +205,7 @@ const TermsAndConditions = () => (
...
                         <li>{it.L('When a force majeure event occurs')}</li>
```
Comment (ShuvoHabib) [inline src/templates/static/legal/metatrader/terms_and_conditions.jsx:210]: Add 'force majeure'

Comment (ShuvoHabib) [inline src/templates/static/legal/metatrader/terms_and_conditions.jsx:210]: Add 'force majeure' text

File: src/templates/static/legal/metatrader/terms_and_conditions.jsx [hunk 5]
```diff
@@ -205,7 +205,7 @@ const TermsAndConditions = () => (
...
                         <li>{it.L('When the market conditions or market movements/volatility are abnormal')}</li>
```
Comment (ShuvoHabib) [inline src/templates/static/legal/metatrader/terms_and_conditions.jsx:211]: Add 'market text'

**Review by kevinw-binary — APPROVED** [review]
